packages/rspack/tests/hotCases/chunk/multi-chunk-single-runtime/changed-file.js

```diff
@@ -0,0 +1,6 @@
+// TODO: remove this file after cache.
```

Comment:
?

Reply:
This file isn't exits at webpack, maybe later need remove.